Earthfile

```diff
@@ -1,4 +1,4 @@
-VERSION --use-copy-include-patterns 0.6
+VERSION --use-copy-include-patterns --try 0.6
```

Comment:
Cannot find this option at docs.earthly.dev. What does it do?

Reply:
> Cannot find this option at docs.earthly.dev. What does it do?

Yeah, it's pretty new. See https://github.com/earthly/earthly/issues/587#issuecomment-1255633387.

Unfortunately it only works for files under 64 kB at the moment. (https://github.com/earthly/earthly/issues/2452) We will only start getting test reports when a new Earthly is released with the fix.